application/backend/src/main/resources/apps/typerefinery/components/widgets/map/variant.html

```diff
@@ -2,8 +2,9 @@
   <div
     id="${model.id}"
     component="${model.componentName}"
-    class="${model.variantClassNames}"
+    class="${model.variantClassNames} card shadow-sm"
```

Comment:
can we add these some other way? otherwise this is hardcoded, maybe have a option for this?